src/XrdSeckrb5/XrdSecProtocolkrb5.cc

```diff
@@ -541,7 +541,7 @@ int XrdSecProtocolkrb5::Authenticate(XrdSecCredentials *cred,
 int XrdSecProtocolkrb5::Init(XrdOucErrInfo *erp, char *KP, char *kfn)
 {
    krb_rc rc;
-   char buff[1024];
+   char buff[2048];
```

Comment:
Out of curiosity, why 2048 ?

Reply:
The problem was that in here:

https://github.com/xrootd/xrootd/blob/fedora-rawhide-fix/src/XrdSeckrb5/XrdSecProtocolkrb5.cc#L585

"Unable to start sequence on the keytab file " gets written to buff, plus 'krb_kt_name', which is up to 1024 (at least that's what the compiler thinks), so the compiler was complaining that 'buff' might be too short. I just picked the next round number ;-), it could have been shorter.